docs/source/torch.compiler_troubleshooting.md

```diff
@@ -780,86 +758,82 @@ Here's a list of useful reproducers, ranked from most to least preferred:
+### Minifier
 
-The minifier is an early ``torch.compile`` tool that, given an FX graph that crashes when we attempt to run or compile it,
+The minifier is an early `torch.compile` tool that, given an FX graph that crashes when we attempt to run or compile it,
 finds a subgraph that also crashes and outputs the code that performs that subgraph's operations.
-Essentially, the minifier finds a minimal repro for a certain class of ``torch.compile``-related crashes.
+Essentially, the minifier finds a minimal repro for a certain class of `torch.compile`-related crashes.
 This assumes that we were able to successfully trace through code.
 
 Unfortunately, most of the time nowadays, the minifier doesn't work as expected, and alternative methods may be necessary.
 This is likely because bugs that can be automatically reproduced in this manner are generally easier to fix
 and have already been addressed, leaving more complex issues that do not reproduce easily.
 However, it is straightforward to attempt using the minifier, so it is worth trying even if it may not succeed.
 
-Instructions for operating the minifier can be found `here <https://pytorch.org/docs/stable/torch.compiler_troubleshooting_old.html>`__.
-If the compiler is crashing, you can set ``TORCHDYNAMO_REPRO_AFTER="dynamo"`` or ``TORCHDYNAMO_REPRO_AFTER="aot"``
-The ``aot`` option is more likely to succeed, although it may not identify the ``AOTAutograd`` issues.  This will generate the ``repro.py`` file which may help to diagnose the problem.
-For accuracy-related issues, consider setting ``TORCHDYNAMO_REPRO_LEVEL=4``. Please note that this may not always successfully identify the problematic subgraph.
+Instructions for operating the minifier can be found [here](https://pytorch.org/docs/stable/torch.compiler_troubleshooting_old.html).
+If the compiler is crashing, you can set `TORCHDYNAMO_REPRO_AFTER="dynamo"` or `TORCHDYNAMO_REPRO_AFTER="aot"`
+The `aot` option is more likely to succeed, although it may not identify the `AOTAutograd` issues. This will generate the `repro.py` file which may help to diagnose the problem.
+For accuracy-related issues, consider setting `TORCHDYNAMO_REPRO_LEVEL=4`. Please note that this may not always successfully identify the problematic subgraph.
 
-Debugging Deeper
-~~~~~~~~~~~~~~~~
+## Debugging Deeper
 
-This section provides tools and techniques for independently debugging ``torch.compile`` issues
-or for gaining a deeper understanding of the ``torch.compile`` stack.
+This section provides tools and techniques for independently debugging `torch.compile` issues
+or for gaining a deeper understanding of the `torch.compile` stack.
 These methods are more involved than those presented above and are used by PyTorch developers regularly
-to debug real ``torch.compile`` issues.
+to debug real `torch.compile` issues.
 
 Below is a high-level overview of the stack:
 
-.. image:: _static/img/dynamo/td_stack.png
+```{image} _static/img/dynamo/td_stack.png```
```

Comment:
Image is not loading: https://docs-preview.pytorch.org/pytorch/pytorch/155351/torch.compiler_troubleshooting.html#debugging-deeper
Maybe try markdown syntax:
![Torch Dynamo Stack](_static/img/dynamo/td_stack.png)